src/types.rs

```diff
@@ -26,6 +34,136 @@ pub struct InternalMetaCallArgs {
     pub input: Vec<u8>,
 }
 
+/// eth-connector initial args
+#[cfg(feature = "contract")]
+pub struct InitCallArgs {
```

Comment:
Borsh-encoded parameter structs should go in `src/parameters.rs`

Reply:
@artob moved